src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Skia.cs

```diff
@@ -158,6 +159,12 @@ internal Size ScrollBarSize
 		public ScrollContentPresenter()
 		{
 			PointerWheelChanged += ScrollContentPresenter_PointerWheelChanged;
+			LayoutUpdated += ScrollContentPresenter_LayoutUpdated; ;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6287442)